corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -82,6 +88,18 @@ def handle(self, app_name, **options):
             self.show_completed_message(app_name)
             return
 
+        self.skip_all = options.get('skip_all')
+        if self.skip_all:
```

Comment:
I might copy `get_confirmation` into this command for the sake of allowing it to check `self.skip_all`, but that's a stylistic comment, not blocking.